### ISSUE TITLE
Usage doesn't get printed

### ISSUE THREAD

**Issue by the-sushi** [apkrieg/gofuck#1]
When I run `gofuck --help` it outputs
`Usage of gofuck:`

With nothing after that.

OSX version 10.12.4 x86_64
Go version: go1.8.1 darwin/amd64

Uname -p: i386
Uname -o: Darwin
Uname -i: MacBookPro12,1
Uname -s: Darwin

**Comment by apkrieg** [comment]
Can you give me some details on the system? If you have Go installed, running `go env` and pasting the output here will help out quite a bit.

**Comment by The-Sushi** [comment]
@apkrieg 
```
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOOS="darwin"
GOPATH="/Users/ender/go"
GORACE=""
GOROOT="/usr/local/go"
GOTOOLDIR="/usr/local/go/pkg/tool/darwin_amd64"
GCCGO="gccgo"
CC="clang"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/k7/wt966sbs13v387mnmgg6bnzc0000gn/T/go-build443093053=/tmp/go-build -gno-record-gcc-switches -fno-common"
CXX="clang++"
CGO_ENABLED="1"
PKG_CONFIG="pkg-config"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
```

**Comment by apkrieg** [comment]
Nevermind, there was never a help page. I'll add one in the next version.

**Comment by the-sushi** [comment]
FTFY

https://github.com/apkrieg/gofuck/pull/2

**Comment by apkrieg** [comment]
Fixed 7b54986
